enhancements/declarative-index-config.md

```diff
@@ -0,0 +1,483 @@
+                        "version": "v1.0.0",
+                        "channel": "alpha",
+                        "upgradeMode": "semver",
+                        "ownedAPIs": [
+                            {
+                                "group": "amq-streams.org.com",
+                                "version": "v1alpha1",
+                                "kind": "amq-streams"
+                            }
+                        ],
+                        "requiredAPIs": [
+                            {
+                                "group": "etcd.database.coreos.com",
+                                "version": "v1alpha1",
+                                "kind": "etcd"
+                            }
+                        ]
+                    }
+                ]
+            },
+            {
+                "name": "etcd",
+                "defaultChannel": "alpha",
+                "icon": "asdfasdfoipuasdfvasdfqer",
+                "bundles": [
+                    {
+                        "path": "quay.io/random/etcdv0.0.1",
+                        "version": "v0.0.1",
+                        "channel": "alpha",
+                        "upgradeMode": "semver",
+                        "ownedAPIs": [
+                            {
+                                "group": "etcd.database.coreos.com",
+                                "version": "v1alpha1",
+                                "kind": "etcd"
+                            }
+                        ],
+                        "requiredAPIs": []
+                    },
+                    {
+                        "path": "quay.io/somethingelse/etcdv0.0.2",
+                        "version": "v0.0.2",
+                        "channel": "alpha",
+                        "upgradeMode": "semver",
+                        "ownedAPIs": [
+                            {
+                                "group": "etcd.database.coreos.com",
+                                "version": "v1alpha1",
+                                "kind": "etcd"
+                            }
+                        ],
+                        "requiredAPIs": []
+                    }
+                ]
+            }
+        ]
+    }
+```
+#### Story 2
+
+As a user with pull permission from the namespace an index is hosted in/as a component of operator-lifecycle-manager, I can query an index for a json/yaml representation of the packages of an index.
+
+`opm index represent pacakges --index=docker.io/my-namespace/community-operators --output=json `
+
+creates a new folder `community-operators` and downloads the package representations that are in the index
+
+```bash=
+$ cd community-operators
+$ tree
+.
+├── etcd.json
+├── amqstreams.json
+
+$ cat etcd.json
+"etcd": {
+        "name": "etcd",
+        "defaultChannel": "alpha",
+        "icon": "asdfasdfoipuasdfvasdfqer",
+        "bundles": [
+            {
+                "path": "quay.io/random/etcdv0.0.1",
```

Comment:
@ecordell since end users are our first class citizens in this context, my preference is to align these properties with CSV fields instead, something that's easily consumable by end users. End users are already accustomed to certain field values when they're creating the bundle(i.e the manifests inside them).